internal/metrics/metrics.go

```diff
@@ -185,6 +187,12 @@ func NewMetrics(registry *prometheus.Registry) *Metrics {
 			},
 			[]string{},
 		),
+		dagRebuildTotal: prometheus.NewCounter(
+			prometheus.CounterOpts{
+				Name: DAGRebuildTotal,
+				Help: "Total number of times DAG has been rebuilt since startup",
```

Comment:
Do you think we should add a startup timestamp as well? This would be unique to the instance of the Contour that started.

Reply:
I think that's a pretty good idea, but we could do it in a followup.